src/acp/rule.ts

```diff
@@ -277,9 +292,9 @@ export function setNoneOfRuleUrl<P extends Policy | ResourcePolicy>(
  * function is still experimental and subject to change, even in a non-major release.
  * ```
  *
- * Get the [[Rule]]'s forbidden by the given [[Policy]]
+ * Get the [[Rule]]s forbidden by the given [[Policy]]
```

Comment:
Same question for "forbidden" or "None Of"

Reply:
2cc92033cba5d2642e628f19235cbc6c712274ea